scripts/ci/run_ci.sh

```diff
@@ -269,6 +269,10 @@ if [ -n "$main_ci" ]; then
 	cat test_file_3.txt test_file_2_in.txt test_file_1_in.txt > test_file.txt
 
 	# Run a subset of tests based on matrix size
+    if [[ $SC == *"sanitycheck"* ]]; then
+    echo "Run Sanitycheck tests"
+    pytest $ZEPHYR_BASE/scripts/tests/sanitycheck
```

Comment:
this needs to generate some results in junit  and be part of the overall test report available thru shippable or whatever we might end up using.

Reply:
Sure, will work on the junit report.

Reply:
Added Junit report as it was done in west and copying it to shippable/testresults.